spec/factories/exception_hunter/errors.rb

```diff
@@ -0,0 +1,28 @@
+FactoryBot.define do
+  factory :exception_hunter_error, class: 'ExceptionHunter::Error' do
```

Comment:
```suggestion
  factory :error, class: 'ExceptionHunter::Error' do
```
I think that should work fine, don't really know if there's a reason to namespace engine factories